console/palette.go

```diff
@@ -2,33 +2,52 @@ package console
 
 import (
 	"fmt"
-	ui "github.com/gizak/termui/v3"
 	"runtime"
+
+	ui "github.com/gizak/termui/v3"
 )
 
+// Theme is a representation of a theme
```

Comment:
Hi @Ivan-Marquez,
Thank you for the contribution. Regarding changes in this file - I know that linter is complaining about lack of comments here, but I'm not a big fun of uninformative comments like `type Theme: Theme is a representation of a theme`

The code is clear without such comments, so I would prefer leave the constants without them.

Reply:
Hello @sqshq ,

Thank you so much for your feedback! I'll be updating this PR with the required changes.

Reply:
@sqshq PR updated with requested changes! And it totally makes sense. The names of the constants are already sufficiently descriptive.

Reply:
thank you!